src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+export enum FilterTypes {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+type Props = {
+  todos: Todo[]
+  onFilterType: (e: any) => void
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  todos,
+  onFilterType,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
```

Comment:
it's not a part of the filter, so rename this component to `Footer` (you may move `nav` to a separate `TodoFilter` component)